test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SigningTestFixture.cs

```diff
@@ -197,11 +197,12 @@ private async Task<CertificateAuthority> CreateDefaultTrustedCertificateAuthorit
             var rootCa = CertificateAuthority.Create(testServer.Url);
             var intermediateCa = rootCa.CreateIntermediateCertificateAuthority();
             var rootCertificate = new X509Certificate2(rootCa.Certificate.GetEncoded());
+            var storeLocation = CertificateStoreUtilities.GetTrustedCertificateStoreLocation();
```

Comment:
Use explicit type (`StoreLocation`) instead of `var`.

Reply:
Thanks! Fixed.